### ISSUE TITLE
get_detectors error in STEM mode

### ISSUE THREAD

**Issue by thomasaarholt** [niermann/temscript#4]
On our Titan G2 in STEM Diffraction mode I get the following error when trying to run `.get_detectors()`.

```python
  File "c:\users\supervisor\documents\python scripts\temscript-master\temscript\
microscope.py", line 192, in get_detectors
    print(info.ShutterModes)
RuntimeError: Expected array bounds of dim 0 to be lower < upper: lower=0, upper=0.
```
For now I have just commented out the code, since I'm not worried about shutters.

**Comment by niermann** [comment]
There was a bug occurring when there is only a single ShutterMode. Should be fixed with commit e3f4cf2562b23b48739e2fc89377d42ab83e00d1. Can you test, whether it works with the new code?

**Comment by thomasaarholt** [comment]
Shall do! :)
